core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -448,6 +448,7 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
         }
       }
     }
+
```

Comment:
As I saw, usually there is a blank line before ending of long methods and classes. You might remove it if you will.